RecoEcal/Configuration/python/RecoEcal_cff.py

```diff
@@ -31,3 +31,11 @@
 _ecalClustersHI += islandClusteringSequence
 for e in [pA_2016, peripheralPbPb, pp_on_XeXe_2017]:
     e.toReplaceWith(ecalClusters, _ecalClustersHI)
+
+from Configuration.Eras.Modifier_ppRef_2017_cff import ppRef_2017
+
+from RecoEcal.EgammaClusterProducers.islandBasicClusters_cfi import islandBasicClusters
+
+_ecalClustersHI_ppRef = ecalClusters.copy()
+_ecalClustersHI_ppRef += islandBasicClusters
+ppRef_2017.toReplaceWith(ecalClusters, _ecalClustersHI_ppRef)
```

Comment:
can this be simply the same as what's used above for pA_2016 etc?

Reply:
This new era does not need the full islandClusteringSequence, but only islandBasicClusters. 
Your suggestion would also work, in that case the other modules in islandClusteringSequence would not be run, since their output will not be used.
I chose to make the customization only for those modules that are going to be run. That is why I made it separate. 
What do you suggest ? If you want, I can move the modification and make part of what is done before.